core-container/entry.sh

```diff
@@ -300,18 +294,46 @@ fi
 if [ "$ESPW_NAME" == "" ]; then
     ESPW_NAME="espw"
 fi
-if [ "$Namespace" == "" ]; then
+if [ "$NAMESPACE" == "" ]; then
     NAMESPACE="default"
 fi
+if [ "$PROVIDER_NAME" == "" ]; then
+    PROVIDER_NAME="default"
+    export PROVIDER_NAME
+fi
+if [ "$PROVIDER_NAMESPACE" == "" ]; then
+    PROVIDER_NAMESPACE=spaceprovider-${PROVIDER_NAME}
+    export PROVIDER_NAMESPACE
+fi
+if [ "$PROVIDER_SECRET_NAME" == "" ]; then
+    PROVIDER_SECRET_NAME=psecret
+fi
+if [ "$PROVIDER_SECRET_NAMESPACE" == "" ]; then
+    PROVIDER_SECRET_NAMESPACE=psecret_namespace
+fi
+if [ "$SPACE_MANAGER_KUBECONFIG" == "" ]; then
+    # if the space_manager_kubeconfig is not set, then we assume the kubestellar
+    # cluster is the space manager cluster.
+    SPACE_MANAGER_KUBECONFIG=$kubestellar_kubeconfig
```

Comment:
I find the usage of the name "kubestellar_kubeconfig" here to be counter-intuitive. When I see "kubestellar" I think first of the multi-cluster configuration management functionality. So I would expect the "kubestellar_kubeconfig" to get me access to the ISes, WDSes, KCS, and MBSes. But the usage in this file seems to be that the "kubestellar_kubeconfig" gets me access to the hosting cluster. Maybe call it "host_kubeconfig" instead?